xml/metering-metering_reconfig.xml

```diff
@@ -116,7 +122,7 @@
     <para>
      To add a softlink for the ceilometer.conf, run:
     </para>
-<screen>ln -s /etc/apache2/sites-available/ceilometer.conf /etc/apache2/sites-enabled</screen>
+<screen>sudo ln -s  /etc/apache2/vhosts.d/ceilometer_modwsgi.vhost /etc/apache2/vhosts.d/ceilometer_modwsgi.conf</screen>
```

Comment:
nit: extra space after -s and before /etc

Reply:
Thx! done